### DISCUSSION TITLE
Arabic Glow-TTS with Phonemes

### DISCUSSION THREAD

**Discussion by AliMokhammad** [coqui-ai/TTS#1335] [General Q&A]
Thank You @all for this great work.
I'm training a custom TTS model for the Arabic Language using Glow-TTS trainer in (TTS/recipes/ljspeech/glow_tts/train_glowtts.py ).
when I set "use_phonemes"= True and phoneme_language="ar", I get an error: text.size = 0. 
I followed the code and it seems like the phonemizer is returning an empty text array.
Are there any recommendations regarding this issue or how to prepare a suitable configuration to train Arabic TTS?

**Comment by king-dahmanus** [comment]
I'm interested so I'd like o ask some questions: 
what dataset are you using?
what is the pipeline/what are the aditional components you hand crafted for the task(a diocretizer, phonemizer etc)?
Have you tried different models other than glow tts?
can you make a repository with your current advancement? Thanks

**Comment by Ryuodan** [comment]
the phonemizer uses https://github.com/rhasspy/gruut so you have to install it.
